scripts/get-shapefiles.py

```diff
@@ -305,18 +338,23 @@ def main():
                 settings[element]['url'].rsplit('/', 1)[-1])
 
             # Creating directory
-            if not os.path.exists(dir_path):
+            try:
                 os.makedirs(dir_path)
+                atexit.register(exit_handler, dir_path)
+            except Exception:
+                pass
 
             # Downloading
+            download_type = -1
             if not args.option_no_download or not os.path.isfile(path_name):
                 print(str(element) + "-1. Downloading '" + dir_name + "'...")
-                file_name = download_file(
+                archive_file_name, download_type = download_file(
                     settings[element]['url'], args.data_dir,
                     args.option_force_update, args.option_no_curl)
 
             # Expanding
-            if not args.option_no_extract:
+            if ((not args.option_no_extract and download_type > 0) or
```

Comment:
[PEP 8 -- Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008/#id19) asserts that "Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation."